.github/actions/tox-run/action.yml

```
@@ -0,0 +1,50 @@
+name: tox-run
+description: All the steps required to run a tox action
+inputs:
+  tox-environment:
+    description: The tox environment to run
+    required: true
+  cache-dependency-path:
+    description: Cache path to pass to setup-python
+    required: false
+    default: pyproject.toml
+  include-prefix:
+    description: The prefix for the C include path
+    required: false
+    default: python
+  python-version:
+    description: Python version that gets installed
+    required: false
+    default: 3.9
+  runner-os:
+    description: The OS the runner runs on
+    default: ubuntu-latest
+    required: false
+  tox-version:
+    description: Version of tox to install
+    required: false
+    default: 4.4.3
+
+runs:
+  using: composite
+  steps:
+    - uses: actions/setup-python@v5
+      with:
+        python-version: "${{ inputs.python-version }}"
+        cache: "pip"
```

```suggestion
        cache: pip
```
I think these quotes can be removed too? Don't think so for the Python version ones because yaml will think it's a float?